docs/samples/bentoml/README.md

```diff
@@ -0,0 +1,149 @@
+# Predict on an InferenceService using BentoML
+
+This example builds an iris classification model API server with BentoML and then deploys
+it as an InferenceService to a KFServing installed cluster.
+
+[BentoML](https://bentoml.org) is an open-source framework for high-performance ML model
```

Comment:
BentoML is an open-source platform for high-performance ML model serving. It makes building production API endpoint for your ML model easy and support all major machine learning training frameworks, including Tensorflow, Keras, PyTorch, XGBoost, scikit-learn etc.

BentoML comes with a high-performance API model server with adaptive micro-batching support, which achieves the advantage of batch processing in online serving. It also provide model management and model deployment functionality, provides an end-to-end model serving workflow, with DevOps best practices baked in.